node-fs-readfile/cat.js

```diff
@@ -0,0 +1,11 @@
+import { readFile } from 'node:fs/promises';
+
+const contentPaths = process.argv.slice(2).map(content =>  `./${content}`);
+
+try {
+  const promises = contentPaths.map(path => readFile(path, 'utf8'))
```

Comment:
I know that `()` around the arrow function argument is optional in this case, but I strongly urge you to get in the habit of using it. It makes the code much easier to read since it makes it obvious that you are looking at an arrow function.

I thought prettier was set up to put those in. Do you have prettier set up?